monitor/monitoring-dam-generator-service/src/main/java/eu/seaclouds/monitor/monitoringDamGeneratorService/Service.java

```diff
@@ -0,0 +1,118 @@
+package eu.seaclouds.monitor.monitoringDamGeneratorService;
+
+import it.polimi.tower4clouds.rules.MonitoringRules;
+
+import java.io.StringWriter;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Marshaller;
+
+import eu.seaclouds.monitor.monitoringDamGenerator.MonitoringDamGenerator;
+import eu.seaclouds.monitor.monitoringDamGenerator.adpParsing.Module;
+
+@Path("/")
+public class Service{
+
+    private static String monitoringManagerIp;
+    private static String monitoringManagerPort;
+
+    private static Map<String, List<Module>> monitoringInfoByApplication=new HashMap<String,List<Module>>();
+    
+    public Service(String monitorUrl,
+            String monitorPort)
+        {
+        monitoringManagerIp = monitorUrl;
+        monitoringManagerPort = monitorPort;
+      
+        }
+    
+    
+    @POST
+    @Path("/damgen")
```

Comment:
you are right, @szenzaro do you agree (since I think you will be the one who will call this API) if I change the pathname to /generateDamMonitoringInfo? @andreaturli is the last a valid proposal?